scripts/data_ingestion/ingest_current_season.py

```
@@ -0,0 +1,220 @@
+
+ROOT = Path(__file__).resolve().parents[1]
+
+
+def run_command(cmd: list[str], description: str) -> bool:
+    """Run a command and return success status."""
+    print(f"\n{'='*60}")
+    print(f"STEP: {description}")
+    print(f"Command: {' '.join(cmd)}")
+    print(f"{'='*60}")
+    
+    result = subprocess.run(cmd, cwd=ROOT, capture_output=True, text=True)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)